doc/comparison.md

```diff
@@ -0,0 +1,25 @@
+# Comparison
+
+| Tool    | Binary | Source Code/LLVM | Available Under-constrained Algorithm |
+|---------|--------|------------------|---------------------------------------|
+| [angr](https://angr.io/)    | Yes    | No*              | Yes*                                   |
+| [Grease](https://github.com/GaloisInc/grease)  | Yes    | Yes              | Yes                                   |
+| [KLEE](https://github.com/klee/klee)  | No     | Yes              | No                                    |
```

Comment:
My understanding of [this blog post](https://blog.trailofbits.com/2019/08/30/binary-symbolic-execution-with-klee-native/) is that KLEE (or, more precisely, KLEE-Native) supports analyzing binaries. Is that enough to bump the "Binary" row entry from a "No" to a "Yes"?

Reply:
Given that i assume it's going through remill id only really bump this to an asterisks about all the llvm tools. Something like any LLVM tool can in theory be run using remill but there are tons of difficults and it basically looks like emulating the binary

Reply:
Fair enough. In that case, should the "Binary" row entry be a "No*" (with an asterisk) instead? I see that you've added a note about KLEE-Native below, which makes me think that it should be.